### PR TITLE
nw.js => 0.13 specific changes to run application

### PR DESCRIPTION
**Pull request by vzamanillo**
Fixes # .

Changes proposed in this pull request:
- nw.js => 0.13 specific app and window changes to run application

I've read the [Contributor License Agreement](/CONTRIBUTING.md#contributor-license-agreement)

### CODE REVIEW

**Comment by vankasteelj** [comment]
It does run without all this. What does your commit change?

File: src/app/language/es.json [hunk 1]
```diff
@@ -1,498 +1,507 @@
 {
-    "External Player": "Reproductor externo",
```
Comment (vankasteelj) [inline src/app/language/es.json]: this file shouldn't be modified.

File: src/app/lib/streamer.js [hunk 2]
```diff
@@ -48,7 +48,7 @@
             // and when the selected lang or default lang is set
             // subtitleDownloading is needed cos this is called every 300ms
 
-            if (stateModel.get('streamInfo').get('torrent').defaultSubtitle && stateModel.get('streamInfo').get('torrent').defaultSubtitle !== 'none' && hasSubtitles && subtitles != null && engine.files[0] && !downloadedSubtitles && !subtitleDownloading) {
+            if (stateModel.get('streamInfo').get('torrent').defaultSubtitle && stateModel.get('streamInfo').get('torrent').defaultSubtitle !== 'none' && hasSubtitles && subtitles !== null && engine.files[0] && !downloadedSubtitles && !subtitleDownloading) {
```
Comment (vankasteelj) [inline src/app/lib/streamer.js]: this is interresting (and useful), but it should be in its own commit, as it's a jshint modification, not needed for nw>13

File: src/app/lib/views/browser/item.js [hunk 3]
```diff
@@ -439,7 +439,7 @@
                                     imdb_id: that.model.get('imdb_id').toString(),
                                 }))
                                 .then(function (res) {
-                                    if (res != null && res.length > 0) {
+                                    if (res !== null && res.length > 0) {
```
Comment (vankasteelj) [inline src/app/lib/views/browser/item.js]: same as above, shouldn't be in this commit, but shipped apart

File: src/app/lib/views/movie_detail.js [hunk 4]
```diff
@@ -107,7 +107,7 @@
                 coverCache = null;
             };
             coverCache.onerror = function () {
-                $('.mcover-image').attr('src', self.model.get('image')).addClass('fadein');
+                $('.mcover-image').attr('src', this.model.get('image')).addClass('fadein');
```
Comment (vankasteelj) [inline src/app/lib/views/movie_detail.js]: doesn't this change the context ? `this` is linked to onerror(), not to onShow()

File: src/app/lib/views/movie_detail.js [hunk 5]
```diff
@@ -183,15 +183,9 @@
...
-                $('.star-container').addClass('hidden');
-                AdvSettings.set('ratingStars', false);
-            } else {
-                $('.number-container').addClass('hidden');
-                $('.star-container').removeClass('hidden');
-                AdvSettings.set('ratingStars', true);
-            }
+            $('.number-container').toggleClass('hidden');
+            $('.star-container').toggleClass('hidden');
+            AdvSettings.set('ratingStars', $('.number-container').hasClass('hidden'));
```
Comment (vankasteelj) [inline src/app/lib/views/movie_detail.js]: This is interresting but should be in its own commit. It isn't linked neither to nw>13, neither to jshint

File: src/app/lib/views/movie_detail.js [hunk 6]
```diff
@@ -232,7 +226,7 @@
                 $('.sub-dropdown').addClass('open');
                 $('.sub-dropdown-arrow').addClass('down');
             }
-            var self = this;
+
```
Comment (vankasteelj) [inline src/app/lib/views/movie_detail.js]: same remark here : if it's unused, then yes, it shouldn't be in the code. But it requires to be in its own separated commit too

File: src/app/lib/views/settings_container.js [hunk 7]
```diff
@@ -597,8 +597,12 @@
...
                 zip.addLocalFile(App.settings['databaseLocation'] + '/' + entry);
             });
-
-            fdialogs.saveFile(zip.toBuffer(), function (err, path) {
+            // https://github.com/exos/node-webkit-fdialogs/issues/9
+            var exportDialog = new fdialogs.FDialog({
+                type: 'save',
+                window: nw.Window.get().window
+            });
+            exportDialog.saveFile(zip.toBuffer(), function (err, path) {
```
Comment (vankasteelj) [inline src/app/lib/views/settings_container.js]: needs another commit

**Comment by vzamanillo** [comment]
@vankasteelj Thank you, I will do a complete review and a new PR.

**Comment by xaiki** [comment]
great ! i was just looking into something similar =)

**Comment by vankasteelj** [comment]
Looks way better, thanks, but it's missing all your gui -> nw changes^^

**Comment by vzamanillo** [comment]
I was preparing a [new PR](https://github.com/butterproject/butter-desktop/pull/453) ^^
